internals/notifier.py

```diff
@@ -864,6 +866,43 @@ def build_email(
     }
 
 
+class IntentToBlinkDevHandler(basehandlers.FlaskHandler):
+  """Submit an intent email directly to blink-dev."""
+  IS_INTERNAL_HANDLER = True
+  EMAIL_TEMPLATE_PATH = 'blink/intent_to_implement.html'
+
+  def process_post_data(self, **kwargs):
+    self.require_task_header()
+    feature_id = self.get_param('feature_id', required=True)
```

Comment:
Future work idea: Maybe we could add API entry points in a second openapi.yaml file for all our tasks.  We don't need to generate code or docs for them, but it would be useful to have typed parameter objects rather than plain python dicts.

Reply:
Or maybe just typedicts to gain type safety while avoiding anyone thinking that that second api.yaml is actually a public API.